angularjs-1.5-basics/src/app/landing-page/landing-page.controller.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var landingPageController = function ($interval, $log, $sce) {
+    var vm = this;
+    vm.allComponents = [
+        {
+            name: 'interval-example',
+            html: $sce.trustAsHtml('<interval-example></interval-example>'),
+            test: $sce.trustAsHtml('<h1>Test</h1>')
+        },
+        {
+            name: 'timeout-example',
+            html: $sce.trustAsHtml('<interval-example></interval-example>'),
+            test: $sce.trustAsHtml('<h1>Test</h1>')
+        }
+
+    ];
+
+    vm.searchText = '';
+    vm.test = '';
+    vm.selectedComponent = '';
+    vm.filteredComponents = [];
+
+    vm.filterComponentList = function () {
+        $log.debug('filtering for new Search Text : ' + vm.selectedComponent);
+
+        if (!vm.selectedComponent) {
+            vm.filteredComponents = _.slice(vm.allComponents, 0, vm.allComponents.length);
+        } else {
+            vm.filteredComponents = _.filter(vm.allComponents, function (component) {
```

Comment:
__[jshint]__
* '_' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->